config/webpack.config.prod.js

```diff
@@ -98,23 +95,6 @@ module.exports = {
     return [autoprefixer];
   },
   plugins: [
-    new HtmlWebpackPlugin({
```

Comment:
The obvious drawback is that `HtmlWebpackPlugin` did some HTML postprocessing and most importantly it automatically added chunk hashes to imported files, which now needs to be done manually on the server.